build.gradle

```diff
@@ -90,7 +90,7 @@ ext {
 	rsocketVersion = '1.0.0-RC6'
 	servletApiVersion = '4.0.1'
 	smackVersion = '4.3.4'
-	springAmqpVersion = project.hasProperty('springAmqpVersion') ? project.springAmqpVersion : '2.2.3.RELEASE'
+	springAmqpVersion = project.hasProperty('springAmqpVersion') ? project.springAmqpVersion : '2.2.4.BUILD-SNAPSHOT'
```

Comment:
Or you will move Spring AMQP 2.2.4 release to February 12th: https://github.com/spring-projects/spring-integration/milestone/23.

Or this PR will be placed on hold until February 26th.

But we can't release the next Spring Integration milestone without proper dependencies.

Thanks for understanding.

Reply:
I moved the release to 2/12.